README.md

```diff
@@ -96,6 +99,99 @@ Considering that you ran the install command above, to remove the operator compl
 ```bash
 make uninstall
 ```
+
+## Automatic Updates
+
+The Nexus Operator is capable of conducting automatic updates within a minor (the `y` in `x.y.z`) when using the community default image (`docker.io/sonatype/nexus3`). In the future Red Hat images will also be supported by this feature.
+> **Note**: custom images will not be supported as there is no guarantee that they follow [semantic versioning](https://semver.org/) and as such, updates within the same minor may be disruptive.
+
+Two fields within the Nexus CR control this behavior:
+
+  - `spec.automaticUpdate.disabled` (*boolean*): Whether the Operator should perform automatic updates. Defaults to `false` (auto updates are enabled). Is set to `false` if `spec.image` is not empty and is different from the default community image.
+  - `spec.automaticUpdate.minorVersion` (*integer*): The Nexus image minor version the deployment should stay in. If left blank and automatic updates are enabled the latest minor is set.
+
+> **Note**: if you wish to set a specific tag when using the default community image you must first disable automatic updates.
+
+> **Important**: a change of minors will *not* be monitored or acted upon as an automatic update. Changing the minor is a manual process initiated by the human operator and as such must be monitored by the human operator.
+ 
+The state of ongoing updates is written to `status.updateConditions`, which can be easily accessed with `kubectl`:
+
+```
+$ kubectl describe nexus
+# (output omitted)
+  Update Conditions:
+    Starting automatic update from 3.26.0 to 3.26.1
+    Successfully updated from 3.26.0 to 3.26.1
+Events:
+  Type    Reason         Age   From    Message
+  ----    ------         ----  ----    -------
+  Normal  UpdateSuccess  59s   nexus3  Successfully updated to 3.26.1
+```
+
+> **Note**: do *not* modify these conditions manually, the Operator reconstructs the update state from these.
+
+### Successful Updates
+
+Once an update finishes successfully, an [Event](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#event-v1-core) is raised. You may view the events from a particular Nexus CR by describing it:
+
+```
+$ kubectl describe <Nexus CR>
+```
+
+Or you may query all events:
+
+```
+$ kubectl get events
+```
+
+A successful update event looks like:
+
+```yaml
+
```

Comment:
Where's the output? :D

Reply:
Oops! I think that's when I found the bug, got interrupted by it lol

I'll add it in a bit, thanks :-)